account_bank_statement_import_auto_reconcile/models/account_bank_statement_import_auto_reconcile.py

```diff
@@ -49,21 +49,19 @@ def _reconcile_move_line(self, statement_line, move_line_id):
         """Helper to reconcile some move line with a bank statement.
         This will create a move to reconcile with and assigns journal_entry_id
         """
-        move = self.env['account.move'].create(
-            self.env['account.bank.statement']._prepare_move(
-                statement_line,
-                (
-                    statement_line.statement_id.name or statement_line.name
-                ) + "/" + str(statement_line.sequence or '')
-            )
+        move = self.env['account.move'].create({
```

Comment:
why not use https://github.com/OCA/OCB/blob/10.0/addons/account/models/account_bank_statement.py#L741 ?

stay as close to what a manual reconciliation does as possible, we want to take advantage of whatever overrides exist from other modules